slides/advanced-02-tuning-hyperparameters.qmd

```diff
@@ -701,18 +707,27 @@ Faceted on the expensiveness of preprocessing used.
 
 <br>
 
-We just tuned the model over the number of trees in the ensemble. 
+*We just tuned the model over the number of trees in the ensemble.* 
 
-What if we set a specific number of trees and tuned the "early stopping" parameters? 
+*What if we set a specific number of trees and tuned the "early stopping" parameters?* 
 
- - How many iterations of worst performance before we stop adding new trees? 
+ - *How many iterations of worst performance before we stop adding new trees?* 
 
-This is an alternate approach to tuning these models. 
+*This is an alternate approach to tuning these models.* 
 
-We could set `trees = 1000` and tune the `stop_iter` parameter. 
+*We could set `trees = 1000` and tune the `stop_iter` parameter.* 
```

Comment:
I'd separate "What is early stopping?" from the "Your turn" slide. Currently the "what's early stopping" slide has a "your turn" header and the actual "your turn" slide is empty apart from the countdown

Reply:
The countdown didn't fit on the page so I added another slide.